### DISCUSSION TITLE
Table "no-data" custom component

### DISCUSSION THREAD

**Discussion by Chriscaracach** [rsuite/rsuite#3327] [Q&A]
Wouldn't it be a good idea if the "no-data" message on <Table> could be customized? That is, the user can send a component with the custom message they prefer (styles and tags too). If the user does not define it, the default message ("No data found") would be displayed.
In our project, we had to create personalized messages for our tables. I like the "No data found", it seems simple and minimal, but isn't flexible enough. Sometimes just saying "No data found" isn't quite accurate. It depends on the business logic.

Best regards

**Comment by SevenOutman** [comment]
Yes Table already supports customizing the message to display via `locale` prop.

```jsx
<Table
  data={data}
  locale={{
    emptyMessage: 'No projects here yet'
  }}
/>
```

Please refer to the docs at https://rsuitejs.com/guide/i18n/#table

---

I'm re-categorizing this discussion as Q&A :)
